samples/FunctionApp/Program.cs

```diff
@@ -17,7 +18,12 @@ static async Task Main(string[] args)
             //<docsnippet_startup>
             var host = new HostBuilder()
                 //<docsnippet_configure_defaults>
-                .ConfigureFunctionsWorkerDefaults()
+                .ConfigureFunctionsWorkerDefaults(builder =>
+                {
+                    builder
+                        .AddApplicationInsights()
```

Comment:
Should we consider exposing a single method (which internally calls these 2) for the convenience of user?

Reply:
Ideally yes... but App Insights has these as two separate concepts. I'd like to try to mirror their usage as much as possible so we can offload the documentation to them. I agree it's kinda odd... but we can review before making this GA.